tests/unittests/test_handler/test_handler_ntp.py

```diff
@@ -268,22 +268,37 @@ def test_ntp_handler_real_distro_ntp_templates(self):
                                                  template_fn=template_fn)
                 content = util.load_file(confpath)
                 if client in ['ntp', 'chrony']:
-                    content_lines = content.splitlines()
-                    expected_servers = [
-                        'server {0} iburst'.format(srv) for srv in servers]
-                    print('distro=%s client=%s' % (distro, client))
-                    for sline in expected_servers:
-                        self.assertIn(sline, content_lines,
-                                      ('failed to render {0} conf'
-                                       ' for distro:{1}'.format(client,
-                                                                distro)))
-                    expected_pools = [
-                        'pool {0} iburst'.format(pool) for pool in pools]
-                    for pline in expected_pools:
-                        self.assertIn(pline, content_lines,
-                                      ('failed to render {0} conf'
-                                       ' for distro:{1}'.format(client,
-                                                                distro)))
+                    if client == 'ntp' and distro == 'alpine':
+                        # NTP for Alpine Linux is Busybox's ntp which
+                        # only supports 'server' lines in its configuration
+                        # file and not with iburst option.
+
+                        content_lines = content.splitlines()
+                        expected_servers = [
+                            'server {0}'.format(srv) for srv in servers]
+                        print('distro=%s client=%s' % (distro, client))
+                        for sline in expected_servers:
+                            self.assertIn(sline, content_lines,
+                                          ('failed to render {0} conf'
+                                           ' for distro:{1}'.format(client,
+                                                                    distro)))
```

Comment:
I'd like to keep the same logic, but just adjust the expected values per distro/client

expected_servers = get_expected_servers(servers, distro, client)
expected_pools = get_expected_pools(pools, distro, client)

And in those functions, adjust the expected values per alpine restrictions

Reply:
Reworked in this way.